components/Radio/Radio.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { SFC, ReactNode, CSSProperties } from "react"
+import { StyledRadio } from "./styledRadio"
+
+type SizeType = "small" | "default"
+
+export interface RadioProps {
+  /** whether checked by default */
+  defaultChecked?: boolean
+  /** called when switch is checked or unchecked */
+  onChange?: (checked: boolean) => void | any
+  /** whether switch is checked */
+  checked?: boolean
+  /** whether switch is disabled */
+  disabled?: boolean
+  /** additional class name of root node */
+  className?: string
+  /** autofocus on mount */
+  autoFocus?: boolean
+  /** Child to render near the checkbox */
+  children?: ReactNode
+  /** Custom styles applied to the root element */
+  style?: CSSProperties
+  /** Custom styles for the checkbox */
+  labelStyle?: CSSProperties
+  /** size of the switch */
+  size?: SizeType
+  /** value of the checkbox */
+  value?: any
+  /** Value of the radio group */
+  groupValue?: any
+  /** Function to set the value of radio on the group */
+  setValue?: (val: any) => void | any
```

Comment:
This should not be needed. it should be handled implicitly by `onChange`, and without the need to pass the value to it...

Reply:
Outdated code.